src/css/mapbox-gl.css

```diff
@@ -518,6 +518,7 @@ a.mapboxgl-ctrl-logo.mapboxgl-compact {
     padding: 0 5px;
     color: #333;
     box-sizing: border-box;
+    white-space: nowrap;
```

Comment:
We have to be mindful of text making the scale wider than the scale calculation assumes, so that the control reflects what is written. Let's double-check this